mongo/README.md

```diff
@@ -11,6 +11,8 @@ Connect MongoDB to Datadog in order to:
 
 You can also create your own metrics using custom `find`, `count` and `aggregate` queries.
 
+**Note**: MongoDB v2.6+ is required for this integration.
```

Comment:
> pymongo version got bumped to 2.8 which requires mongodb 2.6+

1. Seems `MongoDB v2.6+` is required since `pymongo 3.6` 

> Version 3.6 adds support for MongoDB 3.6, drops support for CPython 3.3 (PyPy3 is still supported), and drops support for MongoDB versions older than 2.6.

2. Should we add this link to the *Note*: https://api.mongodb.com/python/current/changelog.html#changes-in-version-3-6-0
Or would that would be more overwhelming than helpful ?

Reply:
More overwhelming than helpful